src/Server/MySQLHandler.h

```diff
@@ -92,9 +92,12 @@ class MySQLHandler : public Poco::Net::TCPServerConnection
     MySQLProtocol::PacketEndpointPtr packet_endpoint;
     std::unique_ptr<Session> session;
 
-    using ReplacementFn = std::function<String(const String & query)>;
-    using Replacements = std::unordered_map<std::string, ReplacementFn>;
-    Replacements replacements;
+    using QueryReplacementFn = std::function<String(const String & query)>;
+    using QueriesReplacements = std::unordered_map<std::string, QueryReplacementFn>;
+    QueriesReplacements queries_replacements;
+
+    using SettingsReplacements = std::unordered_map<std::string, std::string>;
```

Comment:
It is not immediately clear what is mapped to what, could you add a comment to l. 98, e.g.

`/// MySQL setting name --> ClickHouse setting name`